packages/ckb-types/index.d.ts

```diff
@@ -50,7 +50,7 @@ declare namespace CKBComponents {
    */
   export interface CellInput {
     prevOutput: OutPoint
```

Comment:
Rename to `previousOutput` as RPC returns

Reply:
will be renamed in [v0.10.0](https://github.com/nervosnetwork/ckb-sdk-js/pull/113/commits/1e90ee1d95bb4586a881aec60edbc6d73ef7d230#diff-17ce93785ffefb6a3d96e7e79d226f65R54)